web_m2x_options/readme/USAGE.md

```diff
@@ -0,0 +1,106 @@
+## in the field's options dict
+
+`create` *boolean* (Default: depends if user have create rights)
+
+> Whether to display the "Create..." entry in dropdown panel.
+
+`create_edit` *boolean* (Default: depends if user have create rights)
+
+> Whether to display "Create and Edit..." entry in dropdown panel
+
+`m2o_dialog` *boolean* (Default: depends if user have create rights)
+
+> Whether to display the many2one dialog in case of validation error.
+
+`limit` *int* (Default: odoo default value is `8`)
+
+> Number of displayed record in drop-down panel
+
+`search_more` *boolean*
+
+> Used to force disable/enable search more button.
+
+`field_color` *string*
```

Comment:
Isn't it color_field instead of field_color?

Reply:
field_color should work using a xml definition like this:
<field name="partner_id" options="{'limit': 10, 'create': false, 'create_edit': false, 'search_more': true, 'field_color':'type', 'colors':{'contact':'green', 'invoice': 'red', 'delivery': 'blue'}}"/>